### ISSUE TITLE
Tidy Relaxation Schemes

### ISSUE THREAD

**Issue by ccoffrin** [lanl-ansi/ThreePhasePowerModels.jl#83]
Following with the PowerModels convention, relaxation schemes should be fairly generic mathematical programming methods and should not know about PowerModels or specific formulation types.

The relaxation schemes here should be updated to be more generic, possibly with different function names.  Then the correct variant can be called from the relaxation constraint functions.

**Comment by frederikgeth** [comment]
Indeed. I need to dispatch based on being a subtype of `AbstractConicPowerFormulation` somehow. Where should this be accomplished?

**Comment by ccoffrin** [comment]
I don't think you need to dispatch on any type in this case.  These functions should simply reason over matrices of JuMP variables / expressions.  The functions in `*_mx.jl` may dispatch by type and call different relaxation schemes based on the power formulation.

**Comment by frederikgeth** [comment]
ok, I'll take a look at that later this week.
